sqlglot/dialects/databricks.py

```diff
@@ -88,3 +110,6 @@ def generatedasidentitycolumnconstraint_sql(
         ) -> str:
             expression.set("this", True)  # trigger ALWAYS in super class
             return super().generatedasidentitycolumnconstraint_sql(expression)
+
+        def jsonpath_sql(self, expression: exp.JSONPath) -> str:
+            return self.expressions(expression, sep="", flat=True).lstrip(".")
```

Comment:
Why are we `lstrip`ing here?

Reply:
I see. Maybe we can control the quote generation in base instead of overriding?

Reply:
I'm fine with either approach, just a thought.

Reply:
Sounds good, I can define a generator flag along the lines of `QUOTE_JSON_PATH` and set it off in Databricks